contrib/babelfishpg_money/fixeddecimal.c

```diff
@@ -409,9 +409,10 @@ fixeddecimal2str(int64 val, char *buffer,
 	 * By default, fixeddecimal contains 4 digits after decimal but if we only need
 	 * 2 decimal points, remove last two decimal digits, round off remaining value.
 	 */
-	if (fixeddecimal_scale == 2)
+	if (fixeddecimal_scale < 4)
```

Comment:
For < 4 you have commented we will remove 2 decimal points, should it be the scale_factor decimal points?

Reply:
Will address this separately.